packages/teams-js/src/internal/internalAPIs.ts

```diff
@@ -63,13 +63,13 @@ export function isHostClientMobile(): boolean {
  *
  * @internal
  */
-export function isApiSupportedOnMobile(requiredVersion: string = defaultSDKVersionForCompatCheck): SdkError {
+export function isApiSupportedOnMobile(requiredVersion: string = defaultSDKVersionForCompatCheck): never | null {
   if (!isHostClientMobile()) {
     const notSupportedError: SdkError = { errorCode: ErrorCode.NOT_SUPPORTED_ON_PLATFORM };
-    return notSupportedError;
+    throw notSupportedError;
   } else if (!isAPISupportedByPlatform(requiredVersion)) {
```

Comment:
>isAPISupportedByPlatform [](http://example.com/codeflow?start=14&length=24)

This is not a request for you to change this here (unless it's easy) but writing this will help me hold in my head what this function is actually doing (because what this function is doing does not match the name of the function).

This function would be more correctly named something like "isCurrentSDKVersionAtLeast"